compiler/il/OMRILOpCodesEnum.hpp

```diff
@@ -171,6 +171,15 @@
    bxor,     // boolean xor of 2 bytes
    sxor,     // boolean xor of 2 short integers
 
+   fmadd,    // scalar float fused multiply add
+   fmsub,    // scalar float fused multiply subtract
+   fnmadd,   // scalar float fused negative multiply add
+   fnmsub,   // scalar float fused negative multiply subtract
+   dmadd,    // scalar double fused multiply add
+   dmsub,    // scalar double fused multiply subtract
+   dnmadd,   // scalar double fused negative multiply add
+   dnmsub,   // scalar double fused negative multiply subtract
+
```

Comment:
It might better to include something like `(a * b) + c` in the comments, since the order isn't quite obvious from its name.

Reply:
done